docs/products/cassandra/concepts/cross-cluster-replication.rst

```diff
@@ -0,0 +1,142 @@
+
+CCR setup
+'''''''''
+
+To make CCR work on your services, you need a cluster comprising two Apache Cassandra services with CCR enabled. On the cluster, you need to issue the CREATE KEYSPACE request, specifying ``NetworkTopologyStrategy`` as a replication strategy along with desired replication factors.
+
+.. code-block:: bash
+   :caption: Example
+
+   CREATE KEYSPACE test WITH replication =  /
+   {                                        /
+    'class': 'NetworkTopologyStrategy',     /
+    'service-1': 3,                         /
+    'service-2': 3                          /
+   };
+
+Where ``service-1`` and ``service-2`` are the names of Apache Cassandra datacenter, which you can find in the Aiven console.
+
+CCR in action
+'''''''''''''
+
+With CCR enabled and configured, Apache Cassandra replicates each write in the keyspace to both services (datacenters) with an appropriate number of copies as per replication factor.
+
+Active-active model
+  Apache Cassandra uses an active-active model: clients have the choice of reading/writing either from one service or the other.
+
+Consistency level
+  The consistency level regulates how many nodes need to confirm they executed an operation for this operation to be considered successfully completed by the client. You can set up the consistency level to one of the allowed consistency level arguments depending on your needs.
+
+.. topic:: Examples
+
+    * LOCAL_QUORUM consistency level
+        The read is contained within the service you connect to (completes faster).
+    * QUORUM consistency level
+        Replies from nodes of both services are required. The read produces more consistent results but fails if one of the regions is unavailable.
+
+.. seealso::
+
+    For more details on consistency levels for Apache Cassandra, see `CONSISTENCY <https://cassandra.apache.org/doc/4.1/cassandra/tools/cqlsh.html#consistency>`_ in the Apache Cassandra documentation.
+
+Limitations
+-----------
+
+* It is not possible to connect two existing services to become a CCR pair.
+
+.. topic:: But you still can 
+    
+   * Create a CCR pair from scratch or
+   * Add a new region to an existing service (create a new service that replicates from your existing service).
+
+* Enabling CCR on an existing service is only possible if this service has a keyspace that uses ``NetworkTopologyStrategy`` as a replication strategy.
+* it's not possible to connect two existing services to become a CCR pair, you can either create a CCR pair from scratch or add a new region to an existing service (create an empty service that replicates from it)
+* Two CCR services need to share one service plan and the same amount of dynamic disk space.
+* Limited replication configuration
+  * ``SimpleReplicationStrategy`` not supported
+  * Unbalanced ``NetworkTopologyStrategy`` not supported (both CCR peer services need the same replication factor)
```

Comment:
also: the RF must be >= 2, and, if unsupported settings are entered, our automation may change those at any time.